static/templates/settings/bot-settings.handlebars

```diff
@@ -7,6 +7,14 @@
             {{/tr}}
         </p>
 
+        <div class="input-group">
+            <label for="type_of_bot">{{t "Type of Bot" }}:</label>
+            <select name="type_of_bot" id="user_type_of_bot">
+                <option value="{{ custom_bot }}">Custom Bot</option>
+                <option value="{{ incoming_webhook_bot }}">Incoming Webhook Bot</option>
+            </select>
+        </div>
```

Comment:
Does this not require JS changes?

Also maybe s/Type of Bot/Bot type/ and s/Custom Bot/Custom bot/

You should also tag the strings for translation; see http://zulip.readthedocs.io/en/latest/translating.html